oak/server/rust/oak_runtime/src/io.rs

```diff
@@ -89,6 +89,9 @@ impl<T: Decodable> Receiver<T> {
                     })
                 })
                 .and_then(|message| T::decode(&message))
+        } else if read_status[0] == ChannelReadStatus::Orphaned {
```

Comment:
(optional) A `match` may read better here instead of the if / else chain?

Reply:
Done.